docs/custom-modal.md

```diff
@@ -0,0 +1,93 @@
+# Creating a Custom MetaMask Modal in React
+
+When integrating MetaMask into your React application, you might need to provide a custom user interface for certain interactions, such as displaying a modal when MetaMask is not installed. This documentation guides you through the process of creating and using a custom modal within the `MetaMaskProvider`.
```

Comment:
maybe more generic, use web instead of react since we want that to work for any framework